internal/connector/test/integration/features/connector-cluster-api.feature

```diff
@@ -68,12 +74,40 @@ Feature: create a connector
     When I PUT path "/v1/kafka_connector_clusters/${cluster_id}" with json body:
       """
       {
-        "name": "My Cluster Name"
+        "name": "My Cluster Name",
+        "annotations": {
+          "cos.bf2.org/organisation-id": "13640203",
+          "test/key": "test-value"
+        }
       }
       """
     Then the response code should be 204
     And the response should match ""
 
+    # Check invalid annotations update that tries to change organisation-id
```

Comment:
Ideally one would test for the other excluded cases like `reservedDomains` as well? That is also why I was asking about the alternative approach in the above comment.

Reply:
Good catch, I'll add a reservedDomains test as well.